builtin/providers/openstack/resource_openstack_fw_firewall_v1.go

```diff
@@ -121,7 +162,11 @@ func resourceFWFirewallV1Read(d *schema.ResourceData, meta interface{}) error {
 		return fmt.Errorf("Error creating OpenStack networking client: %s", err)
 	}
 
-	firewall, err := firewalls.Get(networkingClient, d.Id()).Extract()
+	result := firewalls.Get(networkingClient, d.Id())
+
+	var firewall Firewall
+	err = result.ExtractInto(&firewall)
```

Comment:
Ah, ok... Well atleast the tests should show us any breakage... 

I guess I could collapse this into a single line, tagging the `ExtractInto` onto the end of the `firewalls.Get`... Thoughts?